include/opt-sched/Scheduler/sched_basic_data.h

```diff
@@ -98,6 +98,9 @@ const int MAX_DEFS_PER_INSTR = 4096;
 // The maximum number of register usages per instruction node.
 const int MAX_USES_PER_INSTR = 4096;
 
+// function for parsing cost function names to enum values
+SPILL_COST_FUNCTION ParseSCFName(std::string name);
```

Comment:
Similarly, `llvm::StringRef` or `const std::string&`

Reply:
will do

Reply:
I fixed this issue in the latest commit